samples/nrf9160/lwm2m_client/sample.yaml

```diff
@@ -1,10 +1,11 @@
 sample:
-  name: nrf LwM2M Sample
```

Comment:
Please don't modify indentation if not needed.

Reply:
Fixed